test/verify/check-services

```diff
@@ -90,7 +98,14 @@ ExecStart=/usr/local/bin/test-service %I
 
         unit_action_btn = '#service-valid .list-group-item:nth-child(1) .btn-group'
         file_action_btn = '#service-valid .list-group-item:nth-child(2) .btn-group'
-
+        # Selects Timer tab
+        b.click('#services-filter :nth-child(4)')
+        b.wait_visible('#services')
+        m.execute("systemctl start test.timer")
+        b.wait_in_text(svc_sel('test.timer'), "Today")
+        b.wait_in_text(svc_sel('test.timer'), "unknown")
```

Comment:
![testservices-testbasic-timer](https://cloud.githubusercontent.com/assets/7456909/16265268/f6af18a8-389b-11e6-8e47-16170d25da77.png)

@dperpeet The test checks if test.timer shows "Today" as part of next run and "unknown" in last trigger time.